src/Scaffolding/VS.Web.CG.Mvc/Blazor/BlazorModel.cs

```diff
@@ -0,0 +1,133 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Scaffolding.Shared;
+using Microsoft.DotNet.Scaffolding.Shared.Project;
+using Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore;
+
+namespace Microsoft.VisualStudio.Web.CodeGenerators.Mvc.Blazor
+{
+    /// <summary>
+    /// Model for Razor templates to create or add to existing Endpoints file for the 'dotnet-aspnet-codegenerator minimalapi' scenario 
+    /// </summary>
+    public class BlazorModel
+    {
+        public BlazorModel(
+           ModelType modelType,
+           string dbContextFullTypeName)
+        {
+            ModelType = modelType ?? throw new ArgumentNullException(nameof(modelType));
```

Comment:
This should be treated as what you did on the PR in WebTools, right? There you removed the exception throwing. Or is this a different scenario?

Reply:
not quite the same scenario, this check is necessary and commandline exception are formatted better.